wagtail/utils/sendfile_streaming_backend.py

```diff
@@ -20,7 +19,7 @@ def sendfile(request, filename, **kwargs):
     ):
         return HttpResponseNotModified()
 
-    response = StreamingHttpResponse(FileWrapper(open(filename, "rb")))
+    response = FileResponse(open(filename, "rb"))
```

Comment:
Is the file never closed after opening?

Reply:
I originally thought this, but the file obj is added to [`_resource_closers`](https://github.com/django/django/blob/main/django/http/response.py#L553), so it's closed once the request fully terminates